test/extended/cluster/clt.go

```diff
@@ -0,0 +1,126 @@
+package cluster
+
+import (
+	"fmt"
+	"sync"
+
+	g "github.com/onsi/ginkgo"
+
+	reale2e "k8s.io/kubernetes/test/e2e"
+	e2e "k8s.io/kubernetes/test/e2e/framework"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+)
+
+var viperConfig string
+
+const (
+	DefaultThreadCount int = 10
+	ChannelSize        int = 1024
+)
+
+var _ = g.Describe("[Feature:Performance][Serial][Fast] Load cluster", func() {
+	defer g.GinkgoRecover()
+	var oc = exutil.NewCLIWithoutNamespace("cl")
+
+	g.BeforeEach(func() {
+		var err error
+		viperConfig = reale2e.GetViperConfig()
+		e2e.Logf("Using config %v", viperConfig)
+		err = ParseConfig(viperConfig, false)
```

Comment:
why are you doing this?

Reply:
To load user configuration like [this one](https://github.com/openshift/svt/blob/master/openshift_scalability/config/golang/nodeVertical.yaml) (any many others) that we use to drive the test.